napari/_qt/layers/qt_image_layer.py

```diff
@@ -74,3 +101,30 @@ def _on_rendering_change(self, event):
                 self.layer.rendering, Qt.MatchFixedString
             )
             self.renderComboBox.setCurrentIndex(index)
+            self._toggle_iso_threhold_visbility()
+
+    def _toggle_iso_threhold_visbility(self):
+        rendering = self.layer.rendering
+        if isinstance(rendering, str):
+            rendering = Rendering(rendering)
+        if rendering == Rendering.ISO:
+            self.isoThesholdSilder.show()
+            self.isoThesholdLabel.show()
+        else:
+            self.isoThesholdSilder.hide()
+            self.isoThesholdLabel.hide()
+
+    def _on_ndisplay_change(self, event):
+        if self.layer.dims.ndisplay == 2:
+            self.isoThesholdSilder.hide()
+            self.isoThesholdLabel.hide()
+            self.renderComboBox.hide()
+            self.renderLabel.hide()
+            self.interpComboBox.show()
+            self.interpLabel.show()
+        else:
+            self.renderComboBox.show()
+            self.renderLabel.show()
+            self.interpComboBox.hide()
+            self.interpLabel.hide()
```

Comment:
I had actually been wondering... does vispy not have different interpolation methods for 3D?  I actually don't hate having a nearest neighbors interpolation for volumetric renderings...

Reply:
i don't think it does, i know i hard coded nearest neighbors a while back, i think for that reason

Reply:
It doesn't have NN as far as I can tell, but rather linear (hence the fringes on labels layers, for example). I agree that NN would be very useful, but we need to modify the vispy shaders for that.

Reply:
ah right, sorry, i was confused